cpp/src/io/csv/csv_gpu.cu

```diff
@@ -351,6 +353,23 @@ __inline__ __device__ cudf::timestamp_ns decode_value(const char *data,
   return milli * 1000000;
 }
 
+// The purpose of this is merely to allow compilation ONLY
+// TODO : make this work for json
```

Comment:
Is this a TODO to add implementation to properly support it or is it a reminder to do the same thing for json?